cluster-operator/src/main/java/io/strimzi/operator/cluster/operator/assembly/KafkaConnectAssemblyOperator.java

```diff
@@ -69,6 +69,7 @@ protected void createOrUpdate(Reconciliation reconciliation, ConfigMap assemblyC
         Future<Void> chainFuture = Future.future();
         deploymentOperations.scaleDown(namespace, connect.getName(), connect.getReplicas())
                 .compose(scale -> serviceOperations.reconcile(namespace, connect.getName(), connect.generateService()))
+                .compose(i -> configMapOperations.reconcile(namespace, connect.getMetricsConfigName(), connect.generateMetricsConfigMap()))
```

Comment:
This is good. But I think you have to also update the methods `delete` and `getResources`. The config map is now part of the deployment and should be deleted / listed.

Reply:
Done